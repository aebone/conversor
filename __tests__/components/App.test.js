import React from 'react';
import { render, screen } from '@testing-library/react';

test('hello jest', () => {
  render(<p>Hello Jest!</p>);
  expect(screen.getByText('Hello Jest!')).toBeInTheDocument();
});
