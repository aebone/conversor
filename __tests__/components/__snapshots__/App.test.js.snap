// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot 1`] = `
Array [
  <header
    className="sc-bdnylx iQLrgo"
    role="banner"
  >
    <span
      className="sc-dlnjPT VMORs"
    >
      +- ×÷
    </span>
    <h1>
      Conversor de Medidas
    </h1>
    <div
      aria-pressed={false}
      className="sc-gtssRu ljXbJc"
      id="change-theme"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex="0"
    >
      <svg
        role="img"
        viewBox="0 0 352 512"
        width={12}
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M96.06 454.35c.01 6.29 1.87 12.45 5.36 17.69l17.09 25.69a31.99 31.99 0 0 0 26.64 14.28h61.71a31.99 31.99 0 0 0 26.64-14.28l17.09-25.69a31.989 31.989 0 0 0 5.36-17.69l.04-38.35H96.01l.05 38.35zM0 176c0 44.37 16.45 84.85 43.56 115.78 16.52 18.85 42.36 58.23 52.21 91.45.04.26.07.52.11.78h160.24c.04-.26.07-.51.11-.78 9.85-33.22 35.69-72.6 52.21-91.45C335.55 260.85 352 220.37 352 176 352 78.61 272.91-.3 175.45 0 73.44.31 0 82.97 0 176zm176-80c-44.11 0-80 35.89-80 80 0 8.84-7.16 16-16 16s-16-7.16-16-16c0-61.76 50.24-112 112-112 8.84 0 16 7.16 16 16s-7.16 16-16 16z"
          fill="currentColor"
        />
      </svg>
    </div>
  </header>,
  <main
    className="sc-hKFyIo bpemQj"
  >
    <article
      className="sc-eCApGN bRgGCF"
    >
      <h2>
        Comprimento
      </h2>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label
          htmlFor="length"
        >
          Valor
        </label>
        <input
          id="length"
          onChange={[Function]}
          type="number"
          value={1}
        />
      </div>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label>
          De
        </label>
        <select
          onChange={[Function]}
          value="mi"
        >
          <option
            value="mm"
          >
            Milímetro (mm)
          </option>
          <option
            value="cm"
          >
            Centímetro (cm)
          </option>
          <option
            value="m"
          >
            Metro (m)
          </option>
          <option
            value="km"
          >
            Quilômetro (km)
          </option>
          <option
            value="in"
          >
            Polegada (in)
          </option>
          <option
            value="yd"
          >
            Jarda (yd)
          </option>
          <option
            value="ft-us"
          >
            Pé - US (ft-us)
          </option>
          <option
            value="ft"
          >
            Pé (ft)
          </option>
          <option
            value="mi"
          >
            Milha (mi)
          </option>
        </select>
      </div>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label>
          Para
        </label>
        <select
          onChange={[Function]}
          value="km"
        >
          <option
            value="mm"
          >
            Milímetro (mm)
          </option>
          <option
            value="cm"
          >
            Centímetro (cm)
          </option>
          <option
            value="m"
          >
            Metro (m)
          </option>
          <option
            value="km"
          >
            Quilômetro (km)
          </option>
          <option
            value="in"
          >
            Polegada (in)
          </option>
          <option
            value="yd"
          >
            Jarda (yd)
          </option>
          <option
            value="ft-us"
          >
            Pé - US (ft-us)
          </option>
          <option
            value="ft"
          >
            Pé (ft)
          </option>
          <option
            value="mi"
          >
            Milha (mi)
          </option>
        </select>
      </div>
      <p
        className="sc-gKAblj jVKqSQ"
      >
        1.6093439485009937
      </p>
    </article>
    <article
      className="sc-eCApGN bRgGCF"
    >
      <h2>
        Velocidade
      </h2>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label
          htmlFor="speed"
        >
          Valor
        </label>
        <input
          id="speed"
          onChange={[Function]}
          type="number"
          value={1}
        />
      </div>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label>
          De
        </label>
        <select
          onChange={[Function]}
          value="m/h"
        >
          <option
            value="m/s"
          >
            Metro por Segundo (m/s)
          </option>
          <option
            value="km/h"
          >
            Quilômetro por Hora (km/h)
          </option>
          <option
            value="m/h"
          >
            Milha por Hora (m/h)
          </option>
          <option
            value="knot"
          >
            Nó (knot)
          </option>
          <option
            value="ft/s"
          >
            Pé por Segundo (ft/s)
          </option>
        </select>
      </div>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label>
          Para
        </label>
        <select
          onChange={[Function]}
          value="km/h"
        >
          <option
            value="m/s"
          >
            Metro por Segundo (m/s)
          </option>
          <option
            value="km/h"
          >
            Quilômetro por Hora (km/h)
          </option>
          <option
            value="m/h"
          >
            Milha por Hora (m/h)
          </option>
          <option
            value="knot"
          >
            Nó (knot)
          </option>
          <option
            value="ft/s"
          >
            Pé por Segundo (ft/s)
          </option>
        </select>
      </div>
      <p
        className="sc-gKAblj jVKqSQ"
      >
        1.609344
      </p>
    </article>
    <article
      className="sc-eCApGN bRgGCF"
    >
      <h2>
        Área
      </h2>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label
          htmlFor="area"
        >
          Valor
        </label>
        <input
          id="area"
          onChange={[Function]}
          type="number"
          value={1}
        />
      </div>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label>
          De
        </label>
        <select
          onChange={[Function]}
          value="m2"
        >
          <option
            value="mm2"
          >
            Milímetro Quadrado (mm2)
          </option>
          <option
            value="cm2"
          >
            Centímetro Quadrado (cm2)
          </option>
          <option
            value="m2"
          >
            Metro Quadrado (m2)
          </option>
          <option
            value="ha"
          >
            Hectare (ha)
          </option>
          <option
            value="km2"
          >
            Quilômetro Quadrado (km2)
          </option>
          <option
            value="in2"
          >
            Polegada Quadrada (in2)
          </option>
          <option
            value="yd2"
          >
            Jarda Quadrada (yd2)
          </option>
          <option
            value="ft2"
          >
            Pé Quadrado (ft2)
          </option>
          <option
            value="ac"
          >
            Acre (ac)
          </option>
          <option
            value="mi2"
          >
            Milha Quadrada (mi2)
          </option>
        </select>
      </div>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label>
          Para
        </label>
        <select
          onChange={[Function]}
          value="mi2"
        >
          <option
            value="mm2"
          >
            Milímetro Quadrado (mm2)
          </option>
          <option
            value="cm2"
          >
            Centímetro Quadrado (cm2)
          </option>
          <option
            value="m2"
          >
            Metro Quadrado (m2)
          </option>
          <option
            value="ha"
          >
            Hectare (ha)
          </option>
          <option
            value="km2"
          >
            Quilômetro Quadrado (km2)
          </option>
          <option
            value="in2"
          >
            Polegada Quadrada (in2)
          </option>
          <option
            value="yd2"
          >
            Jarda Quadrada (yd2)
          </option>
          <option
            value="ft2"
          >
            Pé Quadrado (ft2)
          </option>
          <option
            value="ac"
          >
            Acre (ac)
          </option>
          <option
            value="mi2"
          >
            Milha Quadrada (mi2)
          </option>
        </select>
      </div>
      <p
        className="sc-gKAblj jVKqSQ"
      >
        3.861017848943985e-7
      </p>
    </article>
    <article
      className="sc-eCApGN bRgGCF"
    >
      <h2>
        Tempo
      </h2>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label
          htmlFor="time"
        >
          Valor
        </label>
        <input
          id="time"
          onChange={[Function]}
          type="number"
          value={1}
        />
      </div>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label>
          De
        </label>
        <select
          onChange={[Function]}
          value="d"
        >
          <option
            value="ns"
          >
            Nanosegundo
          </option>
          <option
            value="mu"
          >
            Microsegundo
          </option>
          <option
            value="ms"
          >
            Milisegundo
          </option>
          <option
            value="s"
          >
            Segundo
          </option>
          <option
            value="min"
          >
            Minuto
          </option>
          <option
            value="h"
          >
            Hora
          </option>
          <option
            value="d"
          >
            Dia
          </option>
          <option
            value="week"
          >
            Semana
          </option>
          <option
            value="month"
          >
            Mês
          </option>
          <option
            value="year"
          >
            Ano
          </option>
        </select>
      </div>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label>
          Para
        </label>
        <select
          onChange={[Function]}
          value="h"
        >
          <option
            value="ns"
          >
            Nanosegundo
          </option>
          <option
            value="mu"
          >
            Microsegundo
          </option>
          <option
            value="ms"
          >
            Milisegundo
          </option>
          <option
            value="s"
          >
            Segundo
          </option>
          <option
            value="min"
          >
            Minuto
          </option>
          <option
            value="h"
          >
            Hora
          </option>
          <option
            value="d"
          >
            Dia
          </option>
          <option
            value="week"
          >
            Semana
          </option>
          <option
            value="month"
          >
            Mês
          </option>
          <option
            value="year"
          >
            Ano
          </option>
        </select>
      </div>
      <p
        className="sc-gKAblj jVKqSQ"
      >
        24
      </p>
    </article>
    <article
      className="sc-eCApGN bRgGCF"
    >
      <h2>
        Ângulo
      </h2>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label
          htmlFor="angle"
        >
          Valor
        </label>
        <input
          id="angle"
          onChange={[Function]}
          type="number"
          value={1}
        />
      </div>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label>
          De
        </label>
        <select
          onChange={[Function]}
          value="grad"
        >
          <option
            value="deg"
          >
            Grau (deg)
          </option>
          <option
            value="rad"
          >
            Radiano (rad)
          </option>
          <option
            value="grad"
          >
            Grado (grad)
          </option>
          <option
            value="arcmin"
          >
            Minuto de Arco (arcmin)
          </option>
          <option
            value="arcsec"
          >
            Segundo de Arco (arcsec)
          </option>
        </select>
      </div>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label>
          Para
        </label>
        <select
          onChange={[Function]}
          value="rad"
        >
          <option
            value="deg"
          >
            Grau (deg)
          </option>
          <option
            value="rad"
          >
            Radiano (rad)
          </option>
          <option
            value="grad"
          >
            Grado (grad)
          </option>
          <option
            value="arcmin"
          >
            Minuto de Arco (arcmin)
          </option>
          <option
            value="arcsec"
          >
            Segundo de Arco (arcsec)
          </option>
        </select>
      </div>
      <p
        className="sc-gKAblj jVKqSQ"
      >
        0.015707963267948967
      </p>
    </article>
    <article
      className="sc-eCApGN bRgGCF"
    >
      <h2>
        Massa
      </h2>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label
          htmlFor="mass"
        >
          Valor
        </label>
        <input
          id="mass"
          onChange={[Function]}
          type="number"
          value={1}
        />
      </div>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label>
          De
        </label>
        <select
          onChange={[Function]}
          value="kg"
        >
          <option
            value="mcg"
          >
            Micrograma (mcg)
          </option>
          <option
            value="mg"
          >
            Miligrama (mg)
          </option>
          <option
            value="g"
          >
            Grama (g)
          </option>
          <option
            value="kg"
          >
            Quilograma (kg)
          </option>
          <option
            value="oz"
          >
            Onça (oz)
          </option>
          <option
            value="lb"
          >
            Libra (lb)
          </option>
          <option
            value="mt"
          >
            Tonelada
          </option>
          <option
            value="t"
          >
            Tonelada Curta
          </option>
        </select>
      </div>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label>
          Para
        </label>
        <select
          onChange={[Function]}
          value="lb"
        >
          <option
            value="mcg"
          >
            Micrograma (mcg)
          </option>
          <option
            value="mg"
          >
            Miligrama (mg)
          </option>
          <option
            value="g"
          >
            Grama (g)
          </option>
          <option
            value="kg"
          >
            Quilograma (kg)
          </option>
          <option
            value="oz"
          >
            Onça (oz)
          </option>
          <option
            value="lb"
          >
            Libra (lb)
          </option>
          <option
            value="mt"
          >
            Tonelada
          </option>
          <option
            value="t"
          >
            Tonelada Curta
          </option>
        </select>
      </div>
      <p
        className="sc-gKAblj jVKqSQ"
      >
        2.2046244201837775
      </p>
    </article>
    <article
      className="sc-eCApGN bRgGCF"
    >
      <h2>
        Volume
      </h2>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label
          htmlFor="volume"
        >
          Valor
        </label>
        <input
          id="volume"
          onChange={[Function]}
          type="number"
          value={1}
        />
      </div>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label>
          De
        </label>
        <select
          onChange={[Function]}
          value="l"
        >
          <option
            value="mm3"
          >
            Milímetro Cúbico (mm3)
          </option>
          <option
            value="cm3"
          >
            Centímetro Cúbico (cm3)
          </option>
          <option
            value="ml"
          >
            Mililitro (ml)
          </option>
          <option
            value="cl"
          >
            Centilitro (cl)
          </option>
          <option
            value="dl"
          >
            Decilitro (dl)
          </option>
          <option
            value="l"
          >
            Litro (l)
          </option>
          <option
            value="kl"
          >
            Quilolitro (kl)
          </option>
          <option
            value="m3"
          >
            Metro Cúbico (m3)
          </option>
          <option
            value="km3"
          >
            Quilômetro Cúbico (km3)
          </option>
          <option
            value="tsp"
          >
            Colher de Chá
          </option>
          <option
            value="Tbs"
          >
            Colher de Sopa
          </option>
          <option
            value="in3"
          >
            Polegada Cúbica (in3)
          </option>
          <option
            value="fl-oz"
          >
            Onça Fluída (fl-oz)
          </option>
          <option
            value="cup"
          >
            Copo
          </option>
          <option
            value="pnt"
          >
            Pinta (Pint)
          </option>
          <option
            value="qt"
          >
            Quarto (Quart)
          </option>
          <option
            value="gal"
          >
            Galão
          </option>
          <option
            value="ft3"
          >
            Pé Cúbico (ft3)
          </option>
          <option
            value="yd3"
          >
            Jarda Cúbica (yd3)
          </option>
        </select>
      </div>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label>
          Para
        </label>
        <select
          onChange={[Function]}
          value="ml"
        >
          <option
            value="mm3"
          >
            Milímetro Cúbico (mm3)
          </option>
          <option
            value="cm3"
          >
            Centímetro Cúbico (cm3)
          </option>
          <option
            value="ml"
          >
            Mililitro (ml)
          </option>
          <option
            value="cl"
          >
            Centilitro (cl)
          </option>
          <option
            value="dl"
          >
            Decilitro (dl)
          </option>
          <option
            value="l"
          >
            Litro (l)
          </option>
          <option
            value="kl"
          >
            Quilolitro (kl)
          </option>
          <option
            value="m3"
          >
            Metro Cúbico (m3)
          </option>
          <option
            value="km3"
          >
            Quilômetro Cúbico (km3)
          </option>
          <option
            value="tsp"
          >
            Colher de Chá
          </option>
          <option
            value="Tbs"
          >
            Colher de Sopa
          </option>
          <option
            value="in3"
          >
            Polegada Cúbica (in3)
          </option>
          <option
            value="fl-oz"
          >
            Onça Fluída (fl-oz)
          </option>
          <option
            value="cup"
          >
            Copo
          </option>
          <option
            value="pnt"
          >
            Pinta (Pint)
          </option>
          <option
            value="qt"
          >
            Quarto (Quart)
          </option>
          <option
            value="gal"
          >
            Galão
          </option>
          <option
            value="ft3"
          >
            Pé Cúbico (ft3)
          </option>
          <option
            value="yd3"
          >
            Jarda Cúbica (yd3)
          </option>
        </select>
      </div>
      <p
        className="sc-gKAblj jVKqSQ"
      >
        1000
      </p>
    </article>
    <article
      className="sc-eCApGN bRgGCF"
    >
      <h2>
        Energia
      </h2>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label
          htmlFor="energy"
        >
          Valor
        </label>
        <input
          id="energy"
          onChange={[Function]}
          type="number"
          value={1}
        />
      </div>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label>
          De
        </label>
        <select
          onChange={[Function]}
          value="Wh"
        >
          <option
            value="Wh"
          >
            Watt-hora (Wh)
          </option>
          <option
            value="mWh"
          >
            Miliwatt-hora (mWh)
          </option>
          <option
            value="kWh"
          >
            Quilowatt-hora (kWh)
          </option>
          <option
            value="MWh"
          >
            Megawatt-hora (MWh)
          </option>
          <option
            value="GWh"
          >
            Gigawatt-hora (GWh)
          </option>
          <option
            value="J"
          >
            Joule (J)
          </option>
          <option
            value="kJ"
          >
            Quilojoule (kJ)
          </option>
        </select>
      </div>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label>
          Para
        </label>
        <select
          onChange={[Function]}
          value="J"
        >
          <option
            value="Wh"
          >
            Watt-hora (Wh)
          </option>
          <option
            value="mWh"
          >
            Miliwatt-hora (mWh)
          </option>
          <option
            value="kWh"
          >
            Quilowatt-hora (kWh)
          </option>
          <option
            value="MWh"
          >
            Megawatt-hora (MWh)
          </option>
          <option
            value="GWh"
          >
            Gigawatt-hora (GWh)
          </option>
          <option
            value="J"
          >
            Joule (J)
          </option>
          <option
            value="kJ"
          >
            Quilojoule (kJ)
          </option>
        </select>
      </div>
      <p
        className="sc-gKAblj jVKqSQ"
      >
        3600
      </p>
    </article>
    <article
      className="sc-eCApGN bRgGCF"
    >
      <h2>
        Temperatura
      </h2>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label
          htmlFor="temperature"
        >
          Valor
        </label>
        <input
          id="temperature"
          onChange={[Function]}
          type="number"
          value={1}
        />
      </div>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label>
          De
        </label>
        <select
          onChange={[Function]}
          value="C"
        >
          <option
            value="C"
          >
            Celcius
          </option>
          <option
            value="K"
          >
            Kelvin
          </option>
          <option
            value="F"
          >
            Fahrenheit
          </option>
          <option
            value="R"
          >
            Rankine
          </option>
        </select>
      </div>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label>
          Para
        </label>
        <select
          onChange={[Function]}
          value="F"
        >
          <option
            value="C"
          >
            Celcius
          </option>
          <option
            value="K"
          >
            Kelvin
          </option>
          <option
            value="F"
          >
            Fahrenheit
          </option>
          <option
            value="R"
          >
            Rankine
          </option>
        </select>
      </div>
      <p
        className="sc-gKAblj jVKqSQ"
      >
        33.8
      </p>
    </article>
    <article
      className="sc-eCApGN bRgGCF"
    >
      <h2>
        Frequência
      </h2>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label
          htmlFor="frequency"
        >
          Valor
        </label>
        <input
          id="frequency"
          onChange={[Function]}
          type="number"
          value={1}
        />
      </div>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label>
          De
        </label>
        <select
          onChange={[Function]}
          value="Hz"
        >
          <option
            value="Hz"
          >
            Hertz (Hz)
          </option>
          <option
            value="mHz"
          >
            Milihertz (mHz)
          </option>
          <option
            value="kHz"
          >
            Quilohertz (kHz)
          </option>
          <option
            value="MHz"
          >
            Megahertz (MHz)
          </option>
          <option
            value="GHz"
          >
            Gigahertz (GHz)
          </option>
          <option
            value="THz"
          >
            Terahertz (THz)
          </option>
          <option
            value="rpm"
          >
            Rotação por Minuto (rpm)
          </option>
          <option
            value="deg/s"
          >
            Grau por Segundo
          </option>
          <option
            value="rad/s"
          >
            Radiano por Segundo
          </option>
        </select>
      </div>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label>
          Para
        </label>
        <select
          onChange={[Function]}
          value="mHz"
        >
          <option
            value="Hz"
          >
            Hertz (Hz)
          </option>
          <option
            value="mHz"
          >
            Milihertz (mHz)
          </option>
          <option
            value="kHz"
          >
            Quilohertz (kHz)
          </option>
          <option
            value="MHz"
          >
            Megahertz (MHz)
          </option>
          <option
            value="GHz"
          >
            Gigahertz (GHz)
          </option>
          <option
            value="THz"
          >
            Terahertz (THz)
          </option>
          <option
            value="rpm"
          >
            Rotação por Minuto (rpm)
          </option>
          <option
            value="deg/s"
          >
            Grau por Segundo
          </option>
          <option
            value="rad/s"
          >
            Radiano por Segundo
          </option>
        </select>
      </div>
      <p
        className="sc-gKAblj jVKqSQ"
      >
        1000
      </p>
    </article>
    <article
      className="sc-eCApGN bRgGCF"
    >
      <h2>
        Pressão
      </h2>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label
          htmlFor="pressure"
        >
          Valor
        </label>
        <input
          id="pressure"
          onChange={[Function]}
          type="number"
          value={1}
        />
      </div>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label>
          De
        </label>
        <select
          onChange={[Function]}
          value="Pa"
        >
          <option
            value="Pa"
          >
            Pascal
          </option>
          <option
            value="hPa"
          >
            Hectopascal
          </option>
          <option
            value="kPa"
          >
            Quilopascal
          </option>
          <option
            value="MPa"
          >
            Megapascal
          </option>
          <option
            value="bar"
          >
            Bar
          </option>
          <option
            value="torr"
          >
            Torr
          </option>
          <option
            value="ksi"
          >
            Quilo-libra por polegada quadrada (ksi)
          </option>
          <option
            value="psi"
          >
            Libra por polegada quadrada (psi)
          </option>
        </select>
      </div>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label>
          Para
        </label>
        <select
          onChange={[Function]}
          value="bar"
        >
          <option
            value="Pa"
          >
            Pascal
          </option>
          <option
            value="hPa"
          >
            Hectopascal
          </option>
          <option
            value="kPa"
          >
            Quilopascal
          </option>
          <option
            value="MPa"
          >
            Megapascal
          </option>
          <option
            value="bar"
          >
            Bar
          </option>
          <option
            value="torr"
          >
            Torr
          </option>
          <option
            value="ksi"
          >
            Quilo-libra por polegada quadrada (ksi)
          </option>
          <option
            value="psi"
          >
            Libra por polegada quadrada (psi)
          </option>
        </select>
      </div>
      <p
        className="sc-gKAblj jVKqSQ"
      >
        0.00001
      </p>
    </article>
    <article
      className="sc-eCApGN bRgGCF"
    >
      <h2>
        Digital
      </h2>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label
          htmlFor="digital"
        >
          Valor
        </label>
        <input
          id="digital"
          onChange={[Function]}
          type="number"
          value={1}
        />
      </div>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label>
          De
        </label>
        <select
          onChange={[Function]}
          value="MB"
        >
          <option
            value="b"
          >
            Bit (b)
          </option>
          <option
            value="Kb"
          >
            Kilobit (Kb)
          </option>
          <option
            value="Mb"
          >
            Megabit (Mb)
          </option>
          <option
            value="Gb"
          >
            Gigabit (Gb)
          </option>
          <option
            value="Tb"
          >
            Terabit (Tb)
          </option>
          <option
            value="B"
          >
            Byte (B)
          </option>
          <option
            value="KB"
          >
            Kilobyte (KB)
          </option>
          <option
            value="MB"
          >
            Megabyte (MB)
          </option>
          <option
            value="GB"
          >
            Gigabyte (GB)
          </option>
          <option
            value="TB"
          >
            Terabyte (TB)
          </option>
        </select>
      </div>
      <div
        className="sc-jSFkmK gSIhPO"
      >
        <label>
          Para
        </label>
        <select
          onChange={[Function]}
          value="GB"
        >
          <option
            value="b"
          >
            Bit (b)
          </option>
          <option
            value="Kb"
          >
            Kilobit (Kb)
          </option>
          <option
            value="Mb"
          >
            Megabit (Mb)
          </option>
          <option
            value="Gb"
          >
            Gigabit (Gb)
          </option>
          <option
            value="Tb"
          >
            Terabit (Tb)
          </option>
          <option
            value="B"
          >
            Byte (B)
          </option>
          <option
            value="KB"
          >
            Kilobyte (KB)
          </option>
          <option
            value="MB"
          >
            Megabyte (MB)
          </option>
          <option
            value="GB"
          >
            Gigabyte (GB)
          </option>
          <option
            value="TB"
          >
            Terabyte (TB)
          </option>
        </select>
      </div>
      <p
        className="sc-gKAblj jVKqSQ"
      >
        0.0009765625
      </p>
    </article>
  </main>,
]
`;
